index.html

```diff
@@ -1,534 +1,1010 @@
-    rel="stylesheet" />
-  <link rel="stylesheet" href="style.css" />
-  <link rel="stylesheet" href="queries.css" />
-  <title>Sebastian Gabriel Nagy | Avocat Cluj</title>
+    <script defer="" src="script.js"></script>
+    <script
+      async=""
+      src="https://static.hotjar.com/c/hotjar-3665471.js?sv=6"
+    ></script>
+  </head>
 
-  <script defer src="script.js"></script>
-</head>
-
-<body>
-  <header class="header">
-    <nav class="nav">
-      <img src="img/logo.png" alt="Bankist logo" class="nav__logo" id="logo" designer="Joe" data-version-number="3.0" />
-      <ul class="nav__links">
-        <li class="nav__item">
-          <a class="nav__link" href="#despre">Despre</a>
-        </li>
-        <li class="nav__item">
-          <a class="nav__link" href="#servicii">Servicii</a>
-        </li>
-        <li class="nav__item">
-          <a class="nav__link nav__link--btn btn--show-modal" href="#contact">Contact</a>
-        </li>
-      </ul>
-      <button id="toggleButton-close">
-        <svg class="btn-close__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" id="menu">
-          <path
-            d="M12,7a2,2,0,1,0-2-2A2,2,0,0,0,12,7Zm0,10a2,2,0,1,0,2,2A2,2,0,0,0,12,17Zm0-7a2,2,0,1,0,2,2A2,2,0,0,0,12,10Z">
-          </path>
-        </svg>
-      </button>
-      <ul class="nav__links nav__links-m nav__links-m--close">
-        <li class="nav__item">
-          <a class="nav__link" href="#despre">Despre</a>
-        </li>
-        <li class="nav__item">
-          <a class="nav__link" href="#servicii">Servicii</a>
-        </li>
-        <li class="nav__item">
-          <a class="nav__link nav__link--btn btn--show-modal" href="#contact">Contact</a>
-        </li>
-      </ul>
-    </nav>
-
-    <div class="header__title">
-      <div class="header-container">
-        <h1>
-          Sebastian Gabriel
-          <!-- Green highlight effect -->
-          <span class="highlight">Nagy</span>
-        </h1>
-        <h4>Avocat definitiv Cluj-Napoca</h4>
-        <p>
-          <span class="header__desc">
-            Activitatea noastră este caracterizata de inovare și adaptabilitate la nevoile clientului, transparență,
-            atenție la detalii, experiență și expertiza de înaltă calitate și nu în ultimul rand, disponibilitate în
-            relația cu clienții.
-          </span>
-        </p>
-      </div>
-      <!-- <button class="btn--text btn--scroll-to">Mai mult &DownArrow;</button> -->
-      <button class="btn btn--show-modal">Contactează</button>
-
-      <!--TODO: smaller image, rounded borders -->
-      <img src="img/hero.jpg" class="header__img"
-        alt="Avocat Sebastian Nagy imagine profil | Avocat definitiv Cluj-Napoca" />
-    </div>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <img
+          src="img/logo.png"
+          alt="Bankist logo"
```

Comment:
Please add a valid alt attribute for this image. A good one should be "Sebastian Gabriel Nagy | Avocat Cluj logo".